File: library/src/main/java/su/levenetc/android/textsurface/interfaces/ITextSurfaceAnimation.java
package su.levenetc.android.textsurface.interfaces;

import android.support.annotation.NonNull;

import su.levenetc.android.textsurface.Text;

/**
 * Created by Eugene Levenetc.
 */
public interface ITextSurfaceAnimation extends ISurfaceAnimation {

	void setInitValues(@NonNull Text text);

	Text getText();

}

File: library/src/main/java/su/levenetc/android/textsurface/interfaces/ITextSurfaceAnimation.kt
package su.levenetc.android.textsurface.interfaces

import su.levenetc.android.textsurface.Text

/**
 * Created by Eugene Levenetc.
 */
interface ITextSurfaceAnimation : ISurfaceAnimation {
    val text: Text
    fun setInitValues(text: Text)
}